core/mondoo-azure-security.mql.yaml

```diff
@@ -713,7 +705,9 @@ queries:
         3. Run this query:
 
           ```mql
-          azure.subscription.storage.accounts { containers { properties['publicAccess'] == "None" } }
+          azure.subscription.storage.accounts { properties.AllowBlobPublicAccess == "false" }
```

Comment:
Also using `.all()` queries here for better actionable output.

Reply:
I intentionally left the description as-is to facilitate a better understanding for the community. This approach should help them grasp how to test effectively and enable more informed comparisons. What are your thoughts on this strategy?